priv/templates/services/mentimeter.tpl

```diff
@@ -0,0 +1,3 @@
+{% if id.has_menti and id.has_mentilink %}
```

Comment:
I've missed last time that `has_mentilink` is dependent on the `has_menti` to be enabled and that the latter is only used for this.

Is it expected to have a mentilink defined while also having it disabled?

If not, then I'd suggest removing `has_menti` from the edit page and make it so that if `has_mentilink` is not empty, then the external service is shown, with:
```suggestion
{% if id.has_mentilink|is_defined %}
```